src/PKSim.Infrastructure/ORM/Repositories/RemoteTemplateRepository.cs

```diff
@@ -85,8 +85,16 @@ public async Task UpdateLocalTemplateSummaryFile()
 
       private IReadOnlyList<RemoteTemplate> expressionProfileFor(RemoteTemplate remoteTemplate, ISimulationSubject simulationSubject)
       {
-         //TODO Not implemented yet. It will be done with the profile defined for individual as separate building block
-         return Array.Empty<RemoteTemplate>();
+         //This will for now assume that the expression profile templates are in the same file
+         return simulationSubject.AllExpressionProfiles()
```

Comment:
For now I think this is a good enough assumptions. But I don't think this will work at that stage because simulation subject expression profiles are not set....we'll need to investigate a bit more